### PR TITLE
make mysql streamer work on travisCI

### PR DESCRIPTION
**Pull request by abrarsheikh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - docker
+sudo: required
+addons:
+  apt:
+    packages:
+      - mysql-server-5.6
+      - mysql-client-core-5.6
+      - mysql-client-5.6
+install:
+ # - echo mysql-apt-config mysql-apt-config/select-server select mysql-5.7 | sudo debconf-set-selections
```
Comment (premsantosh) [inline .travis.yml]: Why are all these commented out? If we don't need them can we remove them?

File: config-env-itest-opensource.yaml [hunk 2]
```diff
@@ -35,10 +33,15 @@ schema_blacklist:
 module_env_config:
     - namespace: data_pipeline
       config:
+        load_schematizer_host_and_port_from_smartstack: False
```
Comment (premsantosh) [inline config-env-itest-opensource.yaml]: This can be removed. (Its defaulted to false.)

File: config-opensource.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+team_name: bam
+
+max_delay_allowed_in_seconds: 600
+
+recovery_queue_size: 1500
+
+namespace: dev
+
+container_name: none
```
Comment (premsantosh) [inline config-opensource.yaml]: none?

File: config-opensource.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    -   namespace: clog
+        config:
+            log_stream_name: tmp_replication_handler_logs
+            # Works around https://bitbucket.org/pypy/pypy/issues/2280
+            scribe_errors_to_syslog: false
+        file: acceptance/configs/clog.yaml
+
+    - namespace: data_pipeline
+      config:
+        load_schematizer_host_and_port_from_smartstack: False
```
Comment (premsantosh) [inline config-opensource.yaml]: Same as above.

File: replication_handler/components/mysql_tools.py [hunk 5]
```diff
@@ -47,7 +47,7 @@ def restore_mysql_dump(db_creds, dump_file):
 
 def create_mysql_dump(db_creds, databases):
     temp_file = _get_dump_file()
-    dump_cmd = "mysqldump --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
+    dump_cmd = "mysqldump --set-gtid-purged=OFF --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
```
Comment (premsantosh) [inline replication_handler/components/mysql_tools.py]: Curious as to why this `set-gtid-purged` is set here? Its defaulted to `AUTO` which checks if `GTID` is enabled on the server or not and accordingly [adds or doesn't add set statement to the output](http://dev.mysql.com/doc/refman/5.7/en/mysqldump.html#option_mysqldump_set-gtid-purged).

File: tests/conftest.py [hunk 6]
```diff
@@ -101,6 +101,7 @@ def services_without_repl_handler(dbs):
 
 @pytest.yield_fixture(scope='module')
 def containers(compose_file, services, dbs, replhandler, rbrsource, schematracker):
+    print "using compose_file: {}".format(compose_file)
```
Comment (premsantosh) [inline tests/conftest.py]: Do we still need this?

File: replication_handler/components/mysql_tools.py (hunk 5 above)
Comment (abrarsheikh) [inline replication_handler/components/mysql_tools.py]: i have removed this, and reverted the mysql version to default of travis which i guess is 5.6.

File: tests/conftest.py (hunk 6 above)
Comment (abrarsheikh) [inline tests/conftest.py]: removed

File: config-env-itest-opensource.yaml (hunk 2 above)
Comment (abrarsheikh) [inline config-env-itest-opensource.yaml]: i had tried to remove this, but it was getting set to true from someplace, i am not sure how. Needs some investigation

**Comment by lsterk** [comment]
Closing this out as we're archiving the repo

**Pull request closed without merging**